README.md

```diff
@@ -23,14 +23,25 @@ It does not make a copy. Make sure your code is checked into a source control
 repository like Git and that you have no outstanding changes to commit before
 running this tool.
 
+```sh
+$ cd my-ember-app-or-addon
+$ npx ember-angle-brackets-codemod ./path/of/files/ or ./some**/*glob.hbs
+```
+
+### Running the codemod with Telemetry
```

Comment:
This section should be first, and the recommended path. Folks should only fallback to using without telemetry when the telemetry gathering route does not work.

Reply:
Yes I will shift things around